modulefiles/build_linux_gnu

```diff
@@ -0,0 +1,87 @@
+#%Module
+
+proc ModulesHelp { } {
+   puts stderr "Load the environmental modules, modules for the compilers if present,"
+   puts stderr "specify compilers, path for HPC-stack and SRW directory on Linux systems" 
+   puts stderr "
+}
+
+module-whatis "Loads libraries needed for building SRW on Linux"
+
+if { [module-info mode load] } {
+   puts "ulimit -S -s unlimited;"
```

Comment:
I can confirm that the puts works as expected (ulimit -s is unlimited after module load) on my Linux box. Since these are supposed to be templates, I think it might be a good idea to comment out almost everything and put in a stderr message that tells users to modify the file(s) to suit their system.

Reply:
I think it is the Lmod version that is the problem. On an ubuntu 18.04 with Lmod 6.6, puts simply prints the command to the screen whether you puts to stdout/stderr, while system has the required effect. It was the opposite for macos though.
If the install instructions for Lmod on linux systems lead to the desired behaviour, I guess there will be no problem so I will approve.